README.md

```diff
@@ -52,34 +60,49 @@ Options:
 
 ## annotated spec:
 
+The below example can be compiled by running `cogs gen basic examples/basic.cog.toml`.
+
+Every cog manifest should have a name key that corresponds to a string.
+
 ```toml
-name = "basic example" # every cog manifest should have a name key that corresponds to a string
+name = "basic example"
+```
 
-# key value pairs for a context/ctx are defined under <ctx>.vars
-# try running `cogs gen basic basic.cog.toml` to see what output cogs generates
+Key-value pairs for a context/ctx are defined under `<ctx>.vars`.
+```toml
 [basic.vars]
 var = "var_value"
 other_var = "other_var_value"
+```
+
+If `<var>.path` is given a string value,
+cogs will look for the key name of `<var>` in the file that that corresponds to the `<var>.path` key,
+returning the corresponding value.
 
-# if <var>.path is given a string value,
-# cogs will look for the key name of <var> in the file that that corresponds to the <var>.path key,
-# returning the corresponding value
+```toml
 manifest_var.path = "./test_files/manifest.yaml"
-# try removing manifest_var from "./test_files/manifest.yaml" and see what happens
+```
+
+Try removing `manifest_var` from `./test_files/manifest.yaml` and see what happens.
```

Comment:
Not sure how much we gain in readability by breaking up the code block, however, we do lose out on maintainability since we can no longer copy/paste `basic.cog.toml` into the code block if the file itself changes.

If we _are_ going to break the instructions out of the TOML file, can we number the steps?

 Thoughts?

Reply:
I reverted it for now. I wanted something a little more comprehensive than just an example file, but I'm not exactly sure exactly what that is yet.